product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -0,0 +1,113 @@
+
+### ba003
+
+ba003 is a combination of SYNCHRNOUS_COMMIT for the local group with LAG_CONTROL parameters being met by at least one external node.
+
+Here is an example of defining a ba003 commit scope using the `bdr.add_commit_scope` command:
+
+```sql
+SELECT bdr.add_commit_scope(
+    commit_scope_name := 'ba003_location-a',
+    origin_node_group := 'location-a',
+    rule := 'MAJORITY (location_a) SYNCHRONOUS_COMMIT AND ANY 1 NOT (location_a) LAG_CONTROL (max_lag_size=1000MB, max_commit_delay=4s, max_lag_time=30s)',
+    wait_for_ready := true
+);
+```
+
+The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it is committed. 
+
+The second part of the rule states that at least one node not in `location_a` must keep lag within the LAG_CONTROL parameters (max_lag_size=1000MB, max_lag_time=30s) with respect to its replicating transactions originating from a node in the `origin_node_group` (here location_a), or a commit delays of up to 4 seconds will be used to slow down processing on the originating node to allow for the lagging node to catch up.
+
+
+### ba004
+
+```sql
+SELECT bdr.add_commit_scope(
+    commit_scope_name := 'ba004_location-a',
+    origin_node_group := 'location-a',
+    rule := 'MAJORITY (location_a) SYNCHRONOUS_COMMIT AND ANY 1 NOT (location_a) SYNCHRONOUS_COMMIT',
+    wait_for_ready := true
+);
+```
+
+ba004 says that for all transactions originating from a node in the `origin_node_group` (in the above example `location-a`), a majority of the nodes of `location-a`, and at least one node not in `location-a`, must confirm the transaction before it is committed.
+
+Just as in [ba001](#ba001) and [ba003](#ba003), ba004 uses `SYNCHRONOUS_COMMIT` to keep potential data loss at near zero, as the data is replicated before commit success is signaled to the application. Again, `MAJORITY` is used so that in a three-node scenario in a region, one node can experience failure without compromising the cluster. However, in this case the commit scope also requires confirmation from one node outside the cluster to be confirmed. 
+
+### Setting default_commit_scope
+
+To make a commit_scope the `default_commit_scope` for a node group, use the following command, `bdr.alter_node_group_option`:
+
+```sql
+SELECT bdr.alter_node_group_option(
+  node_group_name := '<node_group_name>',
+  config_key := 'default_commit_scope',
+  config_value := '<commit_scope_name>'
+);
+```
+
```

Comment:
Might be an idea to mention that commit scopes can be applied per transaction and that the BA presets do not stop users creating their own commit scopes which can be applies as needed.

Reply:
cool. Added the first suggestion toward the end and the second suggestion toward the beginning.